.gitignore

```diff
@@ -50,3 +50,6 @@ dump.rdb
 
 # Ignore vendored local databases
 /vendor/*mmdb
+
+# RVM gemset
+.ruby-gemset
```

Comment:
Can you remove this pls?

Reply:
@iHiD : this file is useful for me, in my local development environment, as a rvm user.
This allows me to separate the gems installation for each project I have in my machine.

Usually, when a repo don't have neither .ruby-gemset itself, nor is ignored, I add it to the .gitignore.

Having it on `.gitignore` will prevent myself (or anybody else) to mistakenly push this file in a PR, as it will show in `git status`.

Is it a problem to have this file ignored?

Looking at the contents of `.gitignore`, this file is similar to `dump.rdb` which is a redis backup file, or `.cache` (cached used for docker).

Reply:
This is one example that ignores files for multiple ruby version managers: https://github.com/forem/forem/blob/main/.gitignore#L72-L79

Mastodon have it in the repo itself: https://github.com/mastodon/mastodon/blob/main/.ruby-gemset

Another one that ignores (that I've added): https://github.com/joemasilotti/railsdevs.com/blob/main/.gitignore#L51-L52

Anyway, if it is problematic to have this in `.gitignore` let me know and I will remove it.

Reply:
Hi @heliocola.

For local files that are not created by the code, it's generally better to include them in your own local exclusions list, rather than adding them to the repositories `.gitignore` file. This avoids having infinite different files listed for different editors, sets up. 

Things like `dump.rdb` and `.cache` are created by our codebase, so having them in a `.gitignore` file is useful as it stops everyone having to deal with the same issue. 

See [this section](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) on GitHub docs for more info on this :)

Reply:
It is not the first time I get this reply back.
I don't want to globally ignore this file because I do use it in all projects I manage.
`.gitignore` is the repo based solution to do this.

The repo already have `.gitignore`, `.dockerignore`, `.eslintignore`, `.ignore`, `.prettierignore`, so I find it really interesting that using this file for its purposes would have such a push back.

To be clear: this isn't a criticism in any shape or form. 
But I am always really surprised how much push back I get on adding this entry to `.gitignore`.

Reply:
My opinion is that having this entry in `.gitignore` (and other editors used by contributors) is a way to welcome users with all kinds of tools! :-D

Anyway, I was truly more concerned about the puma major version update than this one! :-D
I will remove it.

Reply:
When your project has 20,000 contributors that each use different IDEs, tools, etc, you quickly realise that if everyone starts configuring things to help with their usecases, you end up in chaos :)

Reply:
I have a different opinion about `.gitignore`, `projects with 20,000 contributors`, and `chaos in large OSS projects`, but I have already removed it as per your request.

Reply:
Just one final comment: this isn't the most important contribution on this PR and neither what motivated me to contribute.

Reply:
> Just one final comment: this isn't the most important contribution on this PR and neither what motivated me to contribute.

Sure. Thanks. I'm just continuing the discussion that was started above. But I'll leave it now if you're done with chatting about it :)

Reply:
Just acting based on the initial reason the PR was closed due to pausing community contributions (ref: https://github.com/exercism/website/pull/6224#issuecomment-1730841435).
So maybe here isn't the proper place for these discussions. That's all.